Update DynamoDB Docs, replaced shell with aws cli command (#251)
quarkiverse/quarkus-amazon-services: docs/modules/ROOT/pages/amazon-dynamodb.adoc

[source,bash,subs="verbatim,attributes"]
----
docker run --publish 4566:8000 amazon/dynamodb-local:1.19.0 -jar DynamoDBLocal.jar -inMemory -sharedDb
----

This starts a DynamoDB instance that is accessible on port `4566`.
You can check if it's working with `aws dynamodb list-tables --endpoint-url http://localhost:4566`.

> ----

Reviewer: ```suggestion
[source,json]
----
```

{
    "TableNames": []
}
----

> You should see this in your terminal. That's it, DynamoDB runs locally.

Reviewer: ```suggestion
You should see the output above in your terminal. That's it, DynamoDB runs locally.
```

We recommend to use the AWS CLI to provision the table:

> === Create the QuakrusFruits Table

Reviewer: ```suggestion
=== Create the QuarkusFruits Table
```
